EN/BattleRes/talk/script/chr012/talk_chr012_50_07.txt

```diff
@@ -46,64 +47,64 @@ Are you the one who summoned me?
 \\@CHARA_EMOVE@<1, -1 , 14>
 /*-auto-*/\\@VOICE_PLAY@<0, "05Roa/Roa05_0082", 100>
 Roa
-Not at all.
+Nah, wasn't me.
 \l
 
 \\@CHARA_IN@< 1, 5, 5, 0, 1>
 /*-auto-*/\\@VOICE_PLAY@<0, "05Roa/Roa05_0083", 100>
 Roa
-I was never interested in the summoning arts. 
-Besides... You've gotta have a little too much 
+I was never interested in the summoning arts.
+Besides... You've gotta have a little too much
 free time to go around poking holes between worlds.
 \l
 
 \\@CHARA_IN@< 1, 5, 4, 0, 1>
 /*-auto-*/\\@VOICE_PLAY@<0, "05Roa/Roa05_0084", 100>
 Roa
-I may not look it, 
-but my schedule is packed to precision.
+And while I may look like I have exactly that,
+I'm afraid my schedule is meticulously packed.
 \l
 
 \\@CHARA_IN@< 1, 5, 2, 0, 1>
 /*-auto-*/\\@VOICE_PLAY@<0, "05Roa/Roa05_0085", 100>
 Roa
-In my mind, foreign elements such as you ought to
-make a hasty exit. The princess is like an 
-easily distracted stray cat, you see.
+If you ask me, foreign elements such as you ought to
+make a hasty exit. The princess is like a cat, you see?
+She gets distracted a bit too easily.
 \l
 
 \\@CHARA_IN@< 1, 5, 1, 0, 1>
 /*-auto-*/\\@VOICE_PLAY@<0, "05Roa/Roa05_0086", 100>
 Roa
-If you two were to fight, 
-my careful preparations would all come to naught.
+I can't have you two fight, or else my careful
+preparations will all be for naught.
 
 \\@CHARA_IN@< 0, 12, 6, 0, 1>
 /*-auto-*/\\@VOICE_PLAY@<0, "12Alt/Alt05_0023", 100>
 Saber
-In other words, you are planning to 
+In other words, you are planning to
 use this city for some sort of ritual.
 \l
 
 \\@CHARA_IN@< 0, 12, 6, 0, 1>
 /*-auto-*/\\@VOICE_PLAY@<0, "12Alt/Alt05_0024", 100>
 Saber
-I still do not know how I was summoned to this 
-place, but now I understand what my duty must be.
+I still have yet to find out why I was summoned here,
+but my duty is unwavering.
 \l
 
 \\@CHARA_IN@< 0, 12, 2, 0, 1>
 /*-auto-*/\\@VOICE_PLAY@<0, "12Alt/Alt05_0025", 100>
 Saber
-No matter the era, no matter the city... 
+No matter the era, no matter the location.
```

Comment:
She very clearly mentions town/city here, so I think it'd be wise to keep it close. I think she's basically referencing her summoning in Fuyuki, hence the use of the word.